apis/fluentbit/v1alpha2/output_types.go

```diff
@@ -138,9 +138,10 @@ func (list OutputList) LoadAsYaml(sl plugins.SecretLoader, depth int) (string, e
 			if item.Spec.RetryLimit != "" {
 				buf.WriteString(fmt.Sprintf("%sretry_limit: %s\n", padding, item.Spec.RetryLimit))
 			}
-			if item.Spec.CustomPlugin != nil && item.Spec.CustomPlugin.Config != "" {
```

Comment:
How are we handling classic config now for custom plugin? I see you have removed it in places and replaced it with yaml config.

Reply:
It will function as before, since it is just removed in the new loadAsYaml function, the old load function is not changed.

Reply:
Oh I see this is func to load as yaml